### DISCUSSION TITLE
How to contact Visual Studio Marketplace for support?

### DISCUSSION THREAD

**Discussion by Logerfo** [microsoft/vscode-discussions#779] [Extension Development QnA] [answered]
I received an e-mail from an Intellectual Property owner asking me to remove references to their IP in one of my Visual Studio Code extensions. How can I contact VS Marketplace to remove the older content, perhaps migrate data like how many times the extension was downloaded and maybe even allow users to upgrade from one extension id to the other one seamlessly? Is there and e-mail address for this kind of thing? The request and the changes I'm making are public at https://github.com/Logerfo/gitlab-notifications/pull/35. Thank you.

**Comment by burkeholland** [comment]
I think @isidorn can help with this. He's gone for a bit but will look when he returns.

**Reply by isidorn** [reply, marked as answer]
@logerfo please send an email to vsmarketplace@microsoft.com
Thank you

**Reply by Logerfo** [reply]
@isidorn I did send an email to this address three weeks ago but didn't get any response :(

**Reply by isidorn** [reply]
Please send another one and @seaniyer can help

**Reply by Logerfo** [reply]
@isidorn I just sent another e-mail, thank you.

**Comment by imbant** [comment]
Hi @Logerfo ,  I have a question about publisher domain name verification. I sent an email to vsmarketplace@microsoft.com yesterday. 
The extension id is `craftLandstudio.ffugclanguage`
I look forward to a reply. Thank you very much.

**Reply by isidorn** [reply]
fyi @seaniyer

**Reply by imbant** [reply]
Hi, I have finished the domain verification via marketplace's help. Thanks a lot

**Comment by mohdmerajmalik** [comment]
Hi @Logerfo , I have a question about publisher domain name verification. My extension was verified and blue tick was coming but i noticed from last week my blue tick was removed and then i added again my domain for verification and still it says its in processing, my extension id is **PurecodeAI.purecode-ai**	and extension name is **PureCode AI**.
I look forward to a reply. Thank you very much.

**Reply by mariaghiondea** [reply]
You were sent a notification about this: The [VS Marketplace policy](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#verify-a-publisher) requires that the extensions need more than 6-month track record of demonstrating extension-relevant content, trust and positive reputation. Since your publisher/extensions have been around for less than 6 months, we request you to resubmit domain verification after 16th February 2025.

**Comment by mohdmerajmalik** [comment]
Hi @seaniyer , I have a question about publisher domain name verification. My extension was verified and blue tick was coming but i noticed from last week my blue tick was removed and then i added again my domain for verification and still it says its in processing, my extension id is PurecodeAI.purecode-ai and extension name is PureCode AI.
I look forward to a reply. Thank you very much.

**Reply by mariaghiondea** [reply]
Please see response to your first question, in short you need to resubmit after February 16th, once you have accumulated enough history in the system.

**Reply by seaniyer** [reply]
Thanks @mariaghiondea for the clarification.

**Comment by stoneC0der** [comment]
Hi @seaniyer I added my domain name for verification, 5 days has passed, probably a week, but still not verified, the message on the publisher details page is still: *Your request to verify the domain is submitted to the marketplace team for processing.
*
Not that I was previously verified, but got failed to renew hosting account on time (I do not know if this was the reason I got unverified, because I created the TXT record in the provider DNS management page, instead of cpanel)

I added the record to both placed this time if I remember correctly. My publisher ID is: Stonec0der

**Reply by mariaghiondea** [reply]
@stoneC0der can you please send email to [vsmarketplace@microsoft.com](mailto:vsmarketplace@microsoft.com)? I usually takes about a week to do domain name verification.

**Comment by egorlem** [comment]
I have the same issue. I submitted a domain verification request for the [publisher](https://marketplace.visualstudio.com/publishers/guezwhoz-schema) about five months ago. This entire time, the status has been: "Your request to verify the domain is submitted to the marketplace team for processing" - with no feedback whatsoever.

**Reply by isidorn** [reply]
fyi @seaniyer

**Reply by mariaghiondea** [reply]
@egorlem I see your request as not approved. This is because your [domain ](https://egorlem.com/)is empty. 
I'm checking if/why you didn't receive a response. Sometimes requests/responses go to junk too.

**Reply by egorlem** [reply]
Thanks for the quick reply. Will it help if I add a brief description of myself and what I do to the page?

**Reply by mariaghiondea** [reply]
The information on the domain needs to be related to the extensions. So, if you can add details about the extensions - what they do, how to use, etc., that would work. You can send another request or email support afterwards.

**Reply by egorlem** [reply]
Thanks for your help

**Comment by precious112** [comment]
Hi @seaniyer 
HI we got an email of your security detection system flagging an old and outdated version of our extension for hardcoded keys,i have sent an email urging the team to review our latest version, the version that was flagged was 0.0.32 where as we're currently at 0.0.36 and the issue has been solved since 0.0.33,i have sent an email to appeal the possible restriction but haven't gotten a reply,the extension id is Sixth.sixth-ai, thank you.

**Reply by mariaghiondea** [reply]
@previous112 at this time we are giving people a heads up and still testing and gathering information to ensure that we aren't causing disruption. The blocking rules aren't applied yet. We will get back to you as soon as we review your information.

**Reply by mariaghiondea** [reply]
I just replied to your email - you aren't going to be affected when we go to Blocking mode.

**Reply by precious112** [reply]
i've gotten the reply ,thank you and have a nice day

**Comment by michelefenu** [comment]
Hi, I'm having the same issue with the verification for my publisher account, [michelefenu](https://marketplace.visualstudio.com/items?itemName=michelefenu.nur-theme-vscode), and my website, [fenu.dev](https://fenu.dev/).

The status is stuck on, "Your request to verify the domain is submitted to the marketplace team for processing."

If I need to add any specific content to the website to get this approved, please let me know :)

Thanks, Michele

**Reply by mariaghiondea** [reply]
@michelefenu when did you submit the request? I'll follow up with our support to ensure that they provide an update.
From looking at your account, it's new - we usually require 6 months of history before approval.

**Reply by michelefenu** [reply]
Hi @mariaghiondea, thank you for the quick response.

I sent the request about three weeks ago. The publisher account is over five years old, and I previously published another extension on the store in September 2020.

Thanks for the follow-up, and have a great day.

**Reply by madhurivadaligithub** [reply]
Hi  @michelefenu 

Thanks for the follow-up. We have updated the status of the domain verification via email. Please review it and reach out to our support alias (VSMarketplace@microsoft.com) for any further details.

**Reply by michelefenu** [reply]
Thanks again. I will update the website and keep support updated.

**Comment by precious112** [comment]
Hi @mariaghiondea ,

I recently received an email stating that a customer reported our team (behind the extension with extension ID Sixth.sixth-ai ) for banning them from our Discord server after they complained about CPU usage in our extension. I’d like to clarify that this report is not accurate.

We did not ban any user from our Discord. In fact, we took down the entire Discord server some time ago because it was partially abandoned and no longer effective as a support channel. Our team decided to consolidate all customer support through email to ensure faster and more consistent communication.

If you or the reporting user try to access the Discord invite link now, you’ll see that the server no longer exists — it has been permanently taken down.

Additionally, the CPU usage issue mentioned in the report is a known concern that has been raised by multiple users. Our engineering team is already working on a fix, which will be included in an upcoming update.

We've sent a reply via email but didn't get any response.

Please let me know if you need any verification or further context regarding this situation.

**Reply by mariaghiondea** [reply]
Thank you for letting me know. 
@madhurivadaligithub FYI

**Comment by WolfRorDev** [comment]
Hi, I’m experiencing the same issue with domain verification for my publisher account WolfRor (wolfror) and my website wolfror.com.

On January 9th, 2026, I received confirmation that the TXT record was successfully validated. I replied the same day with clarification after being informed that the domain was considered not clearly related to the extensions.

I sent a follow-up message on January 21st, 2026, and I have reached out again today, but I have not received any response so far. The verification status remains unchanged.

Could someone please advise whether any additional information or changes are required on my side to complete the verification?

Thank you in advance for your help.

**Reply by mariaghiondea** [reply]
@WolfRorDev thank you for reaching out and for the patience. I'll engage with support to ensure this gets followed up. FYI @madhurivadaligithub
